common/presenters/move-to-card-component.js

```diff
@@ -16,36 +16,54 @@ function _removeEmpty(items, keys) {
   })
 }
 
-module.exports = function moveToCardComponent({ id, reference, person }) {
-  const age = `(${i18n.t('age')} ${filters.calculateAge(person.date_of_birth)})`
-  const meta = [
-    {
-      label: i18n.t('fields::date_of_birth.label'),
-      html: person.date_of_birth
-        ? `${filters.formatDate(person.date_of_birth)} ${age}`
-        : '',
-    },
-    {
-      label: i18n.t('fields::gender.label'),
-      text: person.gender ? person.gender.title : '',
-    },
-  ]
-
-  return {
-    href: `/move/${id}`,
-    title: {
-      text: person.fullname ? person.fullname.toUpperCase() : '',
-    },
-    caption: {
-      text: i18n.t('moves::move_reference', {
-        reference,
-      }),
-    },
-    meta: {
-      items: _removeEmpty(meta, ['text', 'html']),
-    },
-    tags: {
-      items: assessmentToTagList(person.assessment_answers, `/move/${id}`),
-    },
+function moveToCardComponent({ showMeta = true, showTags = true } = {}) {
+  return function item({ id, reference, person = {} }) {
+    const {
+      gender = {},
+      fullname = '',
+      date_of_birth: dateOfBirth,
+      assessment_answers: assessment,
+    } = person
+    const card = {
+      href: `/move/${id}`,
+      title: { text: fullname.toUpperCase() },
+      caption: {
+        text: i18n.t('moves::move_reference', {
+          reference,
+        }),
+      },
+    }
+
+    if (showMeta) {
+      const dateOfBirthLabel = i18n.t('age', {
+        context: 'with_date_of_birth',
+        age: filters.calculateAge(dateOfBirth),
+        date_of_birth: filters.formatDate(dateOfBirth),
```

Comment:
I'd be tempted not to use snake case here. Is there a reason it's not camelCase?

Reply:
I can't remember the exact thought behind it actually as to why the translation are using snakecase instead of camelcase.

For now though and for consistency would be better to keep them the same.

I think one reason was the API would be returning snakecase and we will sometimes use those key values as translations. So rather than have a mix within the translation files we started using snakecase in them.